packages/components/src/navigation/back-button/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Icon, chevronLeft } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { useNavigationContext } from '../context';
+import { MenuBackButtonUI } from '../styles/navigation-styles';
+
+export default function NavigationMenu( props ) {
+	const { backButtonLabel, className, href, onClick, parentMenu } = props;
```

Comment:
Nitpick: since `props` is not used in the component, we could just destructure it in the function parameter.

```jsx
export default function NavigationMenu( { backButtonLabel, className, href, onClick, parentMenu } ) {
```